ecosystem.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  apps : [{
+    name: 'rdr-bridge',
+    script: 'main.py',
+    interpreter: 'python3',
```

Comment:
```suggestion
    interpreter: 'python3 -u',
```
Pythonでは標準でstdoutがバッファされるので、無効化するために`-u`が必要です

Reply:
追加しました